tests/Yaapii.Atoms.Tests/IO/WriterAsOutputTest.cs

```diff
@@ -59,13 +59,15 @@ public void WritesLargeContentToFile()
             ).Value();
 
             //Read from large file and write to output file (make a copy)
+            var filestream = new FileStream(outputPath, FileMode.OpenOrCreate, FileAccess.Write, FileShare.Write);
```

Comment:
Why is it taken to the top?

Reply:
Just for debuging test. to see if the stream closes or stay open by intellisense/immediately-Window